src/google-apis/auth-client.ts

```diff
@@ -62,9 +57,10 @@ var API = 'https://clouderrorreporting.googleapis.com/v1beta1';
  * @property {Object} _logger - the instance-cached logger instance
  */
 export class RequestHandler extends common.Service {
-  private _config: any;
-  private _logger: any;
-  private authClient: any;
+  private _config: Configuration;
+  private _logger: common.Logger;
+  // TODO: Make this more precise
+  private authClient: {getToken: (err: {}) => void;};
```

Comment:
Assuming this is a google auth client, types are in the 1.0 release that's coming today :)

Reply:
Awesome

Reply:
and it's out ;)